lib/resources.js

```diff
@@ -3,9 +3,9 @@
 const resourceNamespace = require('./ResourceNamespace');
 
 module.exports = {
+  Accounts: require('./resources/Accounts'),
   // Support Accounts for consistency, Account for backwards compatibility
   Account: require('./resources/Accounts'),
-  Accounts: require('./resources/Accounts'),
```

Comment:
Why does `Accounts` appear before `Account` now?

Reply:
That was just the simplest way to do the sorting, I think. I don't think it's quite as good, hopefully we can change it back later (was gonna be a bit annoying I think)